README.NL.rst

```diff
@@ -38,7 +38,16 @@ Versie          Release datum   API specificatie
 ==============  ==============  =============================
 latest          n/a             `ReDoc <https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/maykinmedia/objects-api/master/src/objects/api/v2/openapi.yaml>`_,
                                 `Swagger <https://petstore.swagger.io/?url=https://raw.githubusercontent.com/maykinmedia/objects-api/master/src/objects/api/v2/openapi.yaml>`_,
-                                (`verschillen <https://github.com/maykinmedia/objects-api/compare/2.4.3..master#diff-b9c28fec6c3f3fa5cff870d24601d6ab7027520f3b084cc767aefd258cb8c40a>`_)
+                                (`verschillen <https://github.com/maykinmedia/objects-api/compare/3.0.0..master>`_)
```

Comment:
Same comment as here: https://github.com/maykinmedia/objecttypes-api/pull/149/files#r1917934515

Reply:
This changed in the README's in the same way the Objecttypes-API did it (renamed "Version" -> "Application version")